pkgs/data/icons/tela-icon-theme/default.nix

```diff
@@ -36,7 +36,8 @@ stdenvNoCC.mkDerivation rec {
     description = "A flat colorful Design icon theme";
     homepage = "https://github.com/vinceliuice/tela-icon-theme";
     license = licenses.gpl3Only;
-    platforms = platforms.unix;
+    # darwin systems use case-insensitive filesystems that cause hash mismatches
+    platforms = subtractLists platforms.darwin platforms.unix;
```

Comment:
I think it would have been easier to set platforms to linux only.

Reply:
I would prefer to have other unix like systems to still be able to use it